### ISSUE TITLE
add a manual and the ability to hide applications access

### ISSUE THREAD

**Issue by jerome-diver** [micheleg/dash-to-dock#237]
There is no manual aviable, also i don't know where to modify what for be able to hide applications.
So this proposition is to find a way to be possible to add a tab whre you can hide applications.
thank you for read.

**Comment by micheleg** [comment]
Hi, what do you mean by "hiding applications"?

**Comment by jerome-diver** [comment]
dash-to-dock serve to show list of applications and ability for open/call them (exact ?).
Then, by "hide applications", i mean that it could be an idea to be able to hide an application from the application list of dash-to-dock. (i don't know how to explain it more simply)
For example, when my old tant open the dash-to-dock, i would like she doesn't see the "terminal" application GUI and also some other one... but just see what she really use.
Tell me if it is not clear (or if someone who understand could help to explain better...).
The idea could be (for do that) to add a right click option roll list box under the search space for select "config mode" or "normal mode". Then from config mode, for each icon application listed, it could be possible from right click on each to have an option for hide it in normal mode (a check box inside the list in front of label: "hide"). The result will be that this application, when go back in normal use mode, will not be show in the list.

**Comment by micheleg** [comment]
Ok, I undestand now. This is definitely not something which will get implemented in this extension. These per-user settings really look to me they should be performed by creating different user accounts in the system. If there are reason why this is not suitable for your case, I can suggest to install the workspace isolated extension [https://extensions.gnome.org/extension/887/workspace-isolated-dash/]. In this way moving to a new empty workspace will get you a clean dock with only the favorites shown. You can also disable the favorites from the dock settings.
